oozie-to-airflow/tests/converter/test_oozie_parser.py

```diff
@@ -238,6 +238,38 @@ def test_parse_action_node_pig_with_file_and_archive(self):
         self.assertEqual(["myNameNode/test_dir/test.txt#test_link.txt"], p_op.mapper.hdfs_files)
         self.assertEqual(["myNameNode/test_dir/test2.zip#test_zip_dir"], p_op.mapper.hdfs_archives)
 
+    def test_parse_mapreduce_node(self):
+        self.parser.params = {
+            "nameNode": "hdfs://",
+            "dataproc_cluster": "mycluster",
+            "gcp_region": "europe-west3",
+        }
+        node_name = "mr-node"
+        # language=XML
+        xml = """
+<action name='{node_name}'>
+    <map-reduce>
+        <name-node>hdfs://</name-node>
+        <prepare>
+            <delete path="hdfs:///user/mapreduce/examples/apps/mapreduce/output"/>
+        </prepare>
+    </map-reduce>
+    <ok to="end"/>
+    <error to="fail"/>
+</action>
+""".format(
+            node_name=node_name
+        )
+
+        action_node = ET.fromstring(xml)
+        self.parser.parse_action_node(action_node)
+        self.assertIn(node_name, self.parser.workflow.nodes)
+        mr_node = self.parser.workflow.nodes[node_name]
+        self.assertIn("end", mr_node.get_downstreams())
```

Comment:
```suggestion
        self.assertEquals(["end"], mr_node.get_downstreams())
```
I try to use assertes in the first place, when it is not possible, I try to use other assertions. WDYT?  I am happy to talk about it in real life ..

Reply:
I agree, we can use `assertEquals` here. However, I copied this functionality from other, existing tests. Therefore we should update all of them from `assertIn` to `assertEquals`. Maybe a separate PR would be a better idea?

Reply:
Ok. Let's do it in a separate PR.